packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -38,15 +38,14 @@ const pendingTxKeyOf = evidence => {
 /**
  * Get the key for the seenTxs SetStore.
  *
- * The key is a composite of `NobleAddress` and transaction `amount` and not
- * meant to be parsable.
+ * The key is based on `txHash` and not meant to be parsable.
  *
  * @param {CctpTxEvidence} evidence
  * @returns {SeenTxKey}
  */
 const seenTxKeyOf = evidence => {
-  const { txHash, chainId } = evidence;
-  return `seenTx:${JSON.stringify([txHash, chainId])}`;
+  const { txHash } = evidence;
+  return `seenTx:${JSON.stringify([txHash])}`;
```

Comment:
Do we need the `seenTx:` part? At the least, we probably don't need to stringify `[txHash]`, just append `txHash` right?